base-notebook/resources/nbproxy/setup.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 CS GROUP - https://www.csgroup.eu
+# All rights reserved
+# This file is provided under MIT license. See LICENSE file.
+
+import setuptools
+
+setuptools.setup(
+    name="jupyter-proxy",
+    version="0.0.1",
+    url="https://github.com/jupyterhub/jupyter-server-proxy/tree/master/contrib/theia",
```

Comment:
This installation process looks complicated, especially in this repo. Couldn't we build a better extension system, maybe in the repo linked above, so that such an extension could be easily installed with standard Jupyter toolset?